lib/controllers/spree/apple_pay_domain_verification_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Spree
+  class ApplePayDomainVerificationController < Spree::BaseController
+    def show
+      gateway = Spree::Gateway::StripeApplePayGateway.active.first
```

Comment:
```suggestion
      gateway = Spree::Gateway::StripeApplePayGateway.active.last
```
If for some reason (ex paranoia) there will be 2 gateways, and always the oldest one will be used.